process/block/shardblock_test.go

```diff
@@ -1560,12 +1560,15 @@ func TestShardProcessor_CommitBlockStorageFailsForHeaderShouldErr(t *testing.T)
 		RootHash:      rootHash,
 	}
 	body := make(block.Body, 0)
+	wg := sync.WaitGroup{}
+	wg.Add(1)
 	hdrUnit := &mock.StorerStub{
 		GetCalled: func(key []byte) (i []byte, e error) {
 			return marshalizer.Marshal(&block.Header{})
 		},
 		PutCalled: func(key, data []byte) error {
 			wasCalled = true
```

Comment:
please use atomic as well for was called (put called)

Reply:
Done